docs/releases/1.9.txt

```diff
@@ -25,7 +25,15 @@ Python 3.2 and added support for Python 3.5.
 What's new in Django 1.9
 ========================
 
-...
+Running tests in parallel
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :djadmin:`test` command now supports a ``--parallel`` option to run a
```

Comment:
Actually, after checking with Marc, there's magic that makes it work for `django-admin test` automatically.